src/itemCheckbox/ItemCheckbox.tsx

```diff
@@ -64,33 +81,44 @@ export class ItemCheckbox extends Component<ItemCheckboxProps> {
           name={name}
           checked={checked}
           onChange={this.onChange}
+          onFocus={this.onFocus}
+          onBlur={this.onBlur}
           disabled={disabled || isLoading}
         />
         <CheckboxIcon isChecked={checked} isLoading={isLoading} isDisabled={disabled} />
       </Fragment>
     )
 
     return (
-      <StyledItemCheckbox
-        className={cc(['kirk-item-checkbox', className])}
-        leftTitle={labelTitle}
-        leftBody={label}
-        leftAddon={leftAddon}
-        rightTitle={data}
-        rightTitleDisplay={TextDisplayType.SUBHEADERSTRONG}
-        rightBody={dataInfo}
-        /* No a11y issue here
+      <FocusVisibleContext.Consumer>
+        {context => (
+          <StyledItemCheckbox
+            className={cc([
+              'kirk-item-checkbox',
+              {
+                'focus-visible': context && this.state.focus,
+              },
+              className,
+            ])}
+            leftTitle={labelTitle}
+            leftBody={label}
+            leftAddon={leftAddon}
+            rightTitle={data}
+            rightTitleDisplay={TextDisplayType.SUBHEADERSTRONG}
+            rightBody={dataInfo}
+            /* No a11y issue here
           - The input is well wrapped with the label
           - The linter can't access the complex components implementation
         */
-        // eslint-disable-next-line jsx-a11y/label-has-associated-control
-        tag={<label />}
-        rightAddon={checkbox}
-        isClickable={!disabled}
-        disabled={disabled}
-        hasHorizontalSpacing={hasHorizontalSpacing}
-        {...a11yAttrs}
-      />
+            // eslint-disable-next-line jsx-a11y/label-has-associated-control
+            tag={<label />}
+            rightAddon={checkbox}
+            disabled={disabled}
```

Comment:
`isClickable` is removed as it provide the `kirk-item--clickable` class, only use for `ChevronIcon` not used here.
The prop `disabled` already do the job.